handler/openid/flow_hybrid.go

```diff
@@ -109,11 +109,14 @@ func (c *OpenIDConnectHybridHandler) HandleAuthorizeEndpointRequest(ctx context.
 		if err := c.AuthorizeExplicitGrantHandler.CoreStorage.CreateAuthorizeCodeSession(ctx, signature, ar.Sanitize(c.AuthorizeExplicitGrantHandler.GetSanitationWhiteList())); err != nil {
 			return errors.WithStack(fosite.ErrServerError.WithCause(err).WithDebug(err.Error()))
 		}
-
-		resp.AddFragment("code", code)
+		if ar.GetRequestForm().Get("response_mode") == "form_post" {
+			resp.AddForm("code", code)
+		} else {
+			resp.AddFragment("code", code)
+		}
 		ar.SetResponseTypeHandled("code")
 
-		hash, err := c.Enigma.Hash(ctx, []byte(resp.GetFragment().Get("code")))
+		hash, err := c.Enigma.Hash(ctx, []byte(resp.GetCode()))
```

Comment:
I don't think this is correct, it could be that `GetCode()` is not properly set. Please use `code` here!

Reply:
+1